### PR TITLE
Fix Setup Changes

### PR DESCRIPTION
**Pull request by otawfik-ms**

### CODE REVIEW

File: src/fsharp/CompileOptions.fs [hunk 1]
```diff
@@ -910,8 +911,8 @@ let deprecatedFlagsFsc tcConfigB =
...
     CompilerOption("jit-optimize", tagNone, OptionUnit (fun _ -> tcConfigB.optSettings <- { tcConfigB.optSettings with jitOptUser = Some true }), Some(DeprecatedCommandLineOptionNoDescription("--jit-optimize", rangeCmdArgs)), None)
     CompilerOption("no-jit-optimize", tagNone, OptionUnit (fun _ -> tcConfigB.optSettings <- { tcConfigB.optSettings with jitOptUser = Some false }), Some(DeprecatedCommandLineOptionNoDescription("--no-jit-optimize", rangeCmdArgs)), None)
-    CompilerOption("jit-tracking", tagNone, OptionUnit (fun _ -> () ), Some(DeprecatedCommandLineOptionNoDescription("--jit-tracking", rangeCmdArgs)), None)
-    CompilerOption("no-jit-tracking", tagNone, OptionUnit (fun _ -> () ), Some(DeprecatedCommandLineOptionNoDescription("--no-jit-tracking", rangeCmdArgs)), None)
+    CompilerOption("jit-tracking", tagNone, OptionUnit (fun _ -> (tcConfigB.jitTracking <- true) ), Some(DeprecatedCommandLineOptionNoDescription("--jit-tracking", rangeCmdArgs)), None)
+    CompilerOption("no-jit-tracking", tagNone, OptionUnit (fun _ -> (tcConfigB.jitTracking <- false) ), Some(DeprecatedCommandLineOptionNoDescription("--no-jit-tracking", rangeCmdArgs)), None)
```
Comment (dsyme) [inline src/fsharp/CompileOptions.fs]: Thanks for this.  I can't work out why we removed this!  it seems like it is still relevant?

Comment (otawfik-ms) [inline src/fsharp/CompileOptions.fs]: Yes. Debugger was depending on this bit to decide whether the assembly was optimized or not.

File: vsintegration/src/FSharp.ProjectSystem.FSharp/VSPackage.resx [hunk 2]
```diff
@@ -456,10 +456,10 @@
     <value>Customizes the environment to maximize code editor screen space and improve the visibility of F# commands and tool windows.</value>
   </data>
   <data name="ProductDetails" xml:space="preserve">
-    <value>Microsoft Visual F# $(VSGeneralBrandVersion)$(VSGeneralBrandRelease)</value>
+    <value>Microsoft Visual F# 4.1</value>
```
Comment (shyamnamboodiripad) [inline vsintegration/src/FSharp.ProjectSystem.FSharp/VSPackage.resx]: Perhaps create a separate variable for this since its used in multiple places?

Comment (otawfik-ms) [inline vsintegration/src/FSharp.ProjectSystem.FSharp/VSPackage.resx]: AFAIK, we cannot pass msbuild properties to resx files.

File: src/fsharp/CompileOptions.fs (hunk 1 above)
Comment (KevinRansom) [inline src/fsharp/CompileOptions.fs]: The Coreclr and Desktop clr post 2.0 ignore that bit, which is why I original removed the wiring.  We should either:   Always set it, or set it to 1 when we are generating symbols.

**Comment by otawfik-ms** [comment]
@dotnet-bot test this please

**Comment by KevinRansom** [comment]
Looks good.

**Comment by OmarTawfik** [comment]
Fixed #1058

**Pull request merged**